api/v1alpha1/groupversion_info.go

```diff
@@ -28,6 +28,9 @@ var (
 	// GroupVersion is group version used to register these objects
 	GroupVersion = schema.GroupVersion{Group: "ran.openshift.io", Version: "v1alpha1"}
 
+	// SchemeGroupVersion is group version used to register these objects. This is expected by code-gen
+	SchemeGroupVersion = schema.GroupVersion{Group: "ran.openshift.io", Version: "v1"}
```

Comment:
is it intentional to use v1 rather than v1alpha1?

Reply:
Yep should be v1alpha1!